### ISSUE TITLE
Baseline Model Training & Prediction (1 hour)

### ISSUE THREAD

**Issue by rcorrea4009** [rcorrea4009/Capstone_Projects#39]
Train both models on training set
Generate predictions for validation set
Store model objects and predictions for comparison
